go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/imkira/go-interpol v1.1.0 // indirect
+	github.com/klauspost/pgzip v1.2.5
```

Comment:
I think this should probably be the default library (with feature flag to revert) as it works in all cases, and is faster for large files. Licence?

Reply:
Sounds good. Just switched default to this.